.github/workflows/test.yml

```diff
@@ -664,6 +650,10 @@ jobs:
             path: subworkflows/nf-core/fastq_align_bwa
           - profile: conda
             path: subworkflows/nf-core/fasta_newick_epang_gappa
+          - path: modules/nf-core/parabricks/fq2bammeth
+          - path: modules/nf-core/parabricks/fq2bam
+          - path: modules/nf-core/parabricks/applybqsr
```

Comment:
as above

Reply:
I have used the same test.yml from [6881]( https://github.com/nf-core/modules/pull/6881) and just added this module as @mashehu suggested

Reply:
I would not add all the other modules until the other PR is merged - thats what I meant

Reply:
Because that runs all the tests in the CI